src/components/Dropdown/dropdown.module.scss

```diff
@@ -1,44 +1,65 @@
-$dropdown-hover: #d7d6d6;
+$dropdown-hover: #d0e7ff;
 $white: #fff;
 $theme-blue: #1d1283;
 $theme-active: #87d870;
 $black: #000;
 
 .dropdown {
-    width: 180px;
-    background-color: $white;
-    border: 2px solid $black;
-    border-radius: 0.5rem;
+    width: 150px;
+    border-radius: 10px;
     color: $black;
     position: absolute;
-    top: 58px;
-    right: 16px;
+    top: 60px;
+    right: 23px;
```

Comment:
in REM please

Reply:
As i want them to always show 60px from top and 23 px from right, i kept it in absolute units.

Reply:
> As i want them to always show 60px from top and 23 px from right, i kept it in absolute units.

will it not be good enough if we change the units that can adjust according to screen size because for mobile screens 60px would be very big..."you can ignore if it's irrelevant"